data/beta/field/datatype.yaml

```diff
@@ -3,4 +3,4 @@ datatype: string
 field: datatype
 phase: beta
 register: datatype
-text: The data type for constraining a field value.
+text: "The form of the data in a specific field. This could be numerical ('2.50') or text ('abcdef')."
```

Comment:
As a programmer, I find the new description confusing. The phrase "data type" is language I understand. Saying "The form of the data" does not have a meaning for me.

Also we do not have a "numerical" datatype. Suggest we replace "This could be numerical ('2.50') or text ('abcdef')." with "For example, integer or datatime." or remove the examples text entirely.